src/pages/en/sponsors.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Sponsors
+description: A list of Quilt sponsors, as well as information about our sponsorship process in general
+layout: /src/layouts/Page.astro
+---
+
+This page contains a list of all past and present sponsorships that Quilt holds, what they provide us, and instructions for applying to become a Quilt sponsor.
+
+Though Quilt is not and will hopefully never be a financially-driven project, sponsors can help us offset our costs, or provide tools or services that would otherwise be inaccessible to us. To help minimize the risk of issues that can occur when introducing sponsorships into an open-source project like Quilt, we have a detailed processes for dealing with sponsorships, detailed in [the Sponsorships RFC](https://github.com/QuiltMC/rfcs/blob/master/rfc/0061-sponsorships.md).
+
+There are two main types of sponsors, endorsed and non-endorsed. Endorsed sponsors are those that require an endorsement from us, such as their logo on our site homepage. Non-endorsed sponsors do not require an endorsement.
+
+## Current Sponsors
+
+### 1Password (Non-Endorsed)
+1Password is a company that makes password managers and other tools for securely storing credentials. They provide us with a free 1Password Teams account to share infrastructure keys, social media logins, and other sensitive data, as part of their [1Password for Open Source Projects](https://github.com/1Password/1password-teams-open-source) program.
+
+### Starchild Systems (Endorsed)
+Starchild Systems is an Infrastructure-as-a-Service company. They host the [Quilt Forum](https://forum.quiltmc.org) on their infrastructure at no cost to us.
```

Comment:
The forum link should reuse the URLs set in `Site.ts`.
```mdx
---
# ...
---
import site from "@data/Site";

...

### Starchild Systems (Endorsed)
Starchild Systems is an Infrastructure-as-a-Service company. They host the <a href={site.forum}>Quilt Forum</a> on their infrastructure at no cost to us.
```
We should also add the OpenCollective page to `Site.ts` at some point, but I wouldn't force you to do it :>